packages/blade/src/components/Form/FormLabel.tsx

```diff
@@ -104,8 +104,8 @@ const FormLabel = ({
         weight="bold"
       >
         {children}
-        {computedAccessibilityNode}
       </Text>
+      {computedAccessibilityNode}
```

Comment:
this was the change?
<img width="420" alt="Screenshot 2022-12-12 at 1 16 51 PM" src="https://user-images.githubusercontent.com/11384858/206989823-c6458a93-93d9-4c4b-a32e-07a7f0b88367.png">

Reply:
Yeah can definitely make a BIG impact on snapshots too 😆